### ISSUE TITLE
Whether you can add C++ version of the examples or demo for c++ beginner? 

### ISSUE THREAD

**Issue by zhouxs1023** [vurtun/nuklear#188]
For c++ beginner,they may use c++ build samples,so I want  you should increase some c++ project files for  examples .
as below:
![default](https://cloud.githubusercontent.com/assets/905931/17217799/3bc20016-5518-11e6-9d47-ecba1672e0b7.PNG)

**Comment by vurtun** [comment]
Hey thanks for your issue, but I don't understand completely what you mean. Do you mean that I should add additional project files to make it easier to build with windows?

**Comment by zhouxs1023** [comment]
yes,using visual studio to build sample for beginner  in windows

**Comment by zhouxs1023** [comment]
added  C++ version  of  the examples  or demo!

**Comment by vurtun** [comment]
I currently don't have a windows system but I hopefully I have some time next week to add batch and visual studio project.

**Comment by zhouxs1023** [comment]
thank you very much!
